great_expectations/expectations/self_check_util.py

```diff
@@ -1650,24 +1650,38 @@ def is_valid(self):
             return self._is_valid
 
 
-def build_test_backends_list():
-    test_backends = ["pandas"]
-    no_spark = True
-    if not no_spark:
+def build_test_backends_list(
```

Comment:
Can we do a quick pycharm refactor to rename "backends" to "execution_engines" now that we've standardized on that name? If it's more than a quick operation, no need.

Reply:
Let's do this in a separate PR.